icicle/src/vec_ops/vec_ops.cu

```diff
@@ -131,19 +132,19 @@ namespace vec_ops {
   }
 
   template <typename E>
-  cudaError_t mul(const E* vec_a, const E* vec_b, int n, VecOpsConfig& config, E* result)
+  cudaError_t mul(E* vec_a, const E* vec_b, int n, VecOpsConfig& config, E* result)
```

Comment:
why did you remove the consts? should this modify the inputs?

Reply:
It should be in-place so yes